pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -55,7 +55,7 @@ const (
 	PgautoscalerModuleName = "pg_autoscaler"
 	balancerModuleName     = "balancer"
 	balancerModuleMode     = "upmap"
-	metricsPort            = 9283
+	MetricsPort            = 9283
```

Comment:
If the new code moves to this package you wouldn't need to expose these vars and functions.